apps/oi/models.py

```diff
@@ -25,6 +25,8 @@
 
 LANGUAGE_STATS = ['de',]
 
+MONTH_CHOICES = [(i, i) for i in range(1,13)]
```

Comment:
The 2nd element in the tuple (the human-readable name) should be the month name.

Jochen- do we have things set up so that doing something like ugettext('January') will output a properly translated month name?  I confess that I haven't had reason to look at our I18N/L10N in years.